relay/sources/Makefile

```diff
@@ -112,8 +112,11 @@ install: build
 	install -m 755 relay-api/cleanup.sh $(DESTDIR)/opt/rudder/share/relay-api/
 	
 	# rudder packaging
-	install -m 755 rudder-pkg $(DESTDIR)/opt/rudder/bin/
-	install -m 644 docopt.py $(DESTDIR)/opt/rudder/share/python/
+	install -m 755 rudder-pkg/rudder-pkg $(DESTDIR)/opt/rudder/bin/
+	install -m 755 rudder-pkg/makeIndex.sh $(DESTDIR)/opt/rudder/bin/
+	install -m 755 rudder-pkg/rudder-pkg.conf $(DESTDIR)/ $(DESTDIR)/var/rudder/tmp/
+	install -m 755 rudder-pkg/rudder_apt_key.pub $(DESTDIR)/var/rudder/
```

Comment:
We shouldn't name it apt key as it is not linked to apt at all here (and will be very surprising on non debian systems)